### DISCUSSION TITLE
Supporting onnxruntime 1.3

### DISCUSSION THREAD

**Discussion by simonvvd** [pykeio/ort#96] [Q&A] [answered]
Hi,

Do you have any idea of how much effort it'd require to support onnxruntime 1.3? Is it doable at all or is `ort` relying on essential features from newer versions?

NVidia's edge platforms are extremely hard to upgrade when it's possible at all, so we're currently stuck with Jetpack 4, which only allows us to build onnxruntime 1.3... If there's a path to getting that support done, we're willing to invest time in it or sponsor someone to do so.

_Thanks for the amazing work on this binding 🙂 It's such a joy to be able to bypass a lot of NVidia's 💩._

**Comment by decahedron1** [comment, marked as answer]
Hi! It's certainly possible, it'll just take a bit of work since `ort` does rely on some newer features of ONNX Runtime. I would probably have to spin it off into a separate repo and make it available as a git dependency (meaning it can't be used by crates published to crates.io) if that's okay.

**Reply by simonvvd** [reply]
Great! I emailed you at contact@pyke.io 🙂
